src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days < 3) {
+    result = days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    result = days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    result = days * 40 - 50;
+  }
```

Comment:
What are 3, 40, 20, 50 etc.?
Let's create variables for all main values to make your code cleaner and more informative.
You can also immediately return the full price instead of saving it in a variable